xml/System.Runtime.CompilerServices/RuntimeWrappedException.xml

```diff
@@ -78,8 +78,8 @@
         <Parameter Name="thrownObject" Type="System.Object" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="thrownObject">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="thrownObject">The exception getting wrapped.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Runtime.CompilerServices.RuntimeWrappedException " /> that contains the inner exception object.</summary>
```

Comment:
The build had warnings, @Anipik. Looking at the [build Details](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C17%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C201912171732197033-3675%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rx363BmwfPJGKWodd5rcIkjuWcY8kRDAIPZIDQYjv%2Fw%3D&st=2019-12-17T18%3A04%3A58Z&se=2020-01-17T18%3A09%3A58Z&sp=r), at the very end of the page, it says that one of the crefs was invalid:

```[Warning] 1 invalid cross reference(s) "System.Runtime.CompilerServices.RuntimeWrappedException ".```

It seems there's a trailing space. Can you please fix this so we can get it merged?

```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Runtime.CompilerServices.RuntimeWrappedException" /> that contains the inner exception object.</summary>
```